sources/regenerate_config.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cat __INSTALL_DIR__/config.yaml.head > __INSTALL_DIR__/config.yaml
+
+for f in __INSTALL_DIR__/config.yaml.d/*; do
+    if [ -f "$f" ]; then
+        cat "$f" >> __INSTALL_DIR__/config.yaml
```

Comment:
This will automatically add a blank line to the beginning of the file output before appending it to the config.yaml

This way, you don't have to require adding the blank line in the admin documentation. Wdyt?

```suggestion
        cat "$f" | sed '1 s/^\s*\S/\n&/' >> __INSTALL_DIR__/config.yaml
```